package.json

```diff
@@ -4,13 +4,13 @@
   "description": "Nodejs bindings for Secure Reliable Transport SDK",
   "main": "index.js",
   "scripts": {
-    "preinstall": "npm install git-clone && node scripts/build-srt-sdk.js",
-    "install": "node-gyp rebuild -j 8",
-    "rebuild": "node-gyp rebuild",
-    "clean": "node-gyp clean",
+    "install": "npm run build-srt && npm run rebuild",
+    "build-srt": "node scripts/build-srt-sdk.js",
```

Comment:
I might be blind, but when/where does it install the git-clone package now?

Reply:
it is runtime dependency of the whole package here, line 41 of this file here :)

Reply:
normally you should not have to call `npm install <some-dep>` explicitely ever.

the idea here is that you put all the runtime or dev-time deps into the appropriate list inside this file below (as you did for other deps already), and then once the package got installed properly you are ready to use any of them, assuming them in node_modules, or via any require/import module-loader which will resolve them there.

Reply:
Ah yes of course. Is the git-clone added as a dev-dependency in this PR?

Reply:
actually i figure, `git-clone` here already was (before this PR), is and should be a plain dependency (not dev-dependency), so it is in the package dependencies section.

why is this a runtime dependency and not a devDependency, well we are getting confused here because we need these build-toolchain stuff (along with node-gyp/addon-api & friends) at runtime as people install this stuff in their env. dev-dependency is the toolchain we need to build/test things. But, and this is exceptional, we actually build the add-on JIT before runtime. and I think it is a good way to do it so, I think actually that is what you are supposed to do and all node-addons do (I am myself new to the wizardry of building add-ons in fact ... :)) I think it is needed to do so for x-platform and node-version ABI support otherwise one would need to ship a lot of bins at publish time :)